### PR TITLE
Fix java cli tests

### PR DESCRIPTION
**Pull request by DenverM80**
Metadata was not being set correctly (the input hash was being overwritten)

Sync tests needed to be updated for gets because the local last modified is now set by get_object / get_bulk to match the original file

### CODE REVIEW

**Comment by DenverM80** [comment]
Need to update a couple of unit tests in the same manner as the Fixed sync integration tests...

**Comment by rpmoore** [comment]
It looks like this change does not capture the last modified metadata anymore

**Comment by DenverM80** [comment]
updated

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 1]
```diff
@@ -276,39 +276,86 @@ public void getCompletedJobJson() throws Exception {
     @Test
     public void getObjectWithSync() throws Exception {
         final String bucketName = "test_get_object_with_sync";
+        final String objectName = "beowulf.txt";
         try {
             // For a Get with sync, the local file needs to be older than the server copy
-            Util.copyFile("beowulf.txt", Util.RESOURCE_BASE_NAME, Util.DOWNLOAD_BASE_NAME);
+            Util.copyFile(objectName, Util.RESOURCE_BASE_NAME, Util.DOWNLOAD_BASE_NAME);
+            File file = new File(Util.DOWNLOAD_BASE_NAME + "/" + objectName);
```
Comment (berkeleysquare) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: An OS-agnostic way to handle the pathing?

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 2]
```diff
@@ -276,39 +276,86 @@ public void getCompletedJobJson() throws Exception {
...
-            assertThat(response.getMessage(), is("SUCCESS: No need to sync beowulf.txt"));
+    @Test
+    public void getObjectWithSyncNoUpdate() throws Exception {
+        final String bucketName = "test_get_object_with_sync_no_update";
+        final String objectName = "beowulf.txt";
+        try {
+            // get_object sets the last_modified property to that of the original file, so we need to spoof for comparison
+            Util.copyFile(objectName, Util.RESOURCE_BASE_NAME, Util.DOWNLOAD_BASE_NAME);
+            File newFile = new File(Util.DOWNLOAD_BASE_NAME + "/" + objectName);
+            DateTime now = new DateTime();
```
Comment (RachelTucker) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: missing final

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 3]
```diff
@@ -276,39 +276,86 @@ public void getCompletedJobJson() throws Exception {
...
 
             final Arguments args = new Arguments(new String[]{"--http", "-c", "get_bulk", "-b", bucketName,
                     "-d", Util.DOWNLOAD_BASE_NAME, "--sync"});
             CommandResponse response = Util.command(client, args);
-            assertThat(response.getMessage(), is("SUCCESS: Synced all the objects from test_get_object to ." + File.separator + "." + File.separator + "output"));
+            assertThat(response.getMessage(), is("SUCCESS: Synced all the objects from " + bucketName + " to ." + File.separator + "." + File.separator + "output"));
+
+            // get_bulk sets the last_modified property to that of the original file, so we need to spoof for comparison
+            final File destFolder = new File(Util.DOWNLOAD_BASE_NAME);
+            DateTime now = new DateTime();
```
Comment (RachelTucker) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: missing final

Comment (DenverM80) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: I thought i set this to be an error in IntelliJ, must be per project.  thanks, will fix

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java (hunk 1 above)
Comment (DenverM80) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: will change to File.separator

**Comment by DenverM80** [comment]
updated

**Comment by RachelTucker** [comment]
Looks good

**Pull request merged**